### PR TITLE
HURUmap Zimbabwe

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Create a HURUmap instance for Zimbabwe

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshots
screencapture-localhost-8000-2018-06-12-10_30_24
screencapture-localhost-8000-profiles-province-15-matabeleland-north-2018-10-02-11_03_26

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
To confirm @phillipahereza , this is just a map with:
 

- country and provinces
- no regions, districts or wards
- no census data

**Comment by phillipahereza** [comment]
@kilemensi yes, but don't merge as yet

**Comment by kilemensi** [comment]
Alright @phillipahereza , can I assign it back to you so I know it's not waiting on me?

**Comment by DavidLemayian** [comment]
@kilemensi + @phillipahereza perhaps to make it easier to know it's WIP please add `[WIP]` at the beginning of the title?

**Comment by phillipahereza** [comment]
@kilemensi please review

File: hurumap_zw/settings.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        'children': ['province'],
+    },
+    'province': {
+        'plural': 'provinces',
+        'children': ['district'],
+    },
+    'district': {
+        'plural': 'districts',
+        'children': [],
+    },
```
Comment (kilemensi) [inline hurumap_zw/settings.py:51]: I see `geo/ward.topojson` below, shouldn't ward also be a level here?

File: hurumap_zw/settings.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                              'postgresql://hurumap_zw:hurumap_zw@localhost/hurumap_zw')
+DJANGO_SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY',
+                                   'this is some not so secret key but..')
+DEBUG = os.environ.get('DJANGO_DEBUG', True)
+DJANGO_SETTINGS_MODULE = os.environ.get('DJANGO_SETTINGS_MODULE',
+                                        'hurumap_zw.settings')
+
+DATABASES = {
+    'default': dj_database_url.parse(DATABASE_URL),
+}
```
Comment (kilemensi) [inline hurumap_zw/settings.py]: Do not replace the whole `DATABASES` object, replace only the part we need. i.e.
```python
DATABASES['default'] = dj_database_url.parse( ...
```

**Review by kilemensi — CHANGES_REQUESTED** [review]

**Comment by kilemensi** [comment]
PS: Lets create and add Google Analytics id for Zimbabwe as well @phillipahereza .

File: hurumap_zw/settings.py (hunk 1 above)
Comment (phillipahereza) [reply hurumap_zw/settings.py:51]: @kilemensi I removed it for now, the data he gave me doesn't go that far down, I could easily add them back if needed

**Review by kilemensi — APPROVED** [review]

**Pull request merged**